ydb/src/grpc_wrapper/raw_table_service/value/value_ydb.rs

```diff
@@ -160,6 +174,22 @@ impl TryFrom<crate::Value> for RawTypedValue {
     }
 }
 
+#[repr(C)]
+pub(crate) union DecimalUnion {
```

Comment:
It is first usage of unsafe in the SDK.

Is other way for split i128 to u64 parts exists?